cartography/intel/aws/ec2/instances.py

```diff
@@ -246,6 +246,55 @@ def load_ec2_instances(
                     ).consume()  # TODO see issue 170
 
             load_ec2_instance_network_interfaces(neo4j_session, instance, update_tag)
+            sync_ec2_instance_ebs_volumes(neo4j_session, instance, current_aws_account_id, update_tag)
+
+
+@timeit
+def sync_ec2_instance_ebs_volumes(
```

Comment:
This is an unnecessary layer of abstraction. Please move the two lines (256-7) into the calling function at line 249.

Reply:
Done